src/training_strategies.jl

```diff
@@ -312,19 +312,19 @@ such that the total number of sampled points is equivalent to the given samples
 ## Positional Arguments
 
 * `weights`: A vector of weights that should sum to 1, representing the proportion of samples at each interval.
-* `samples`: the total number of samples that we want, across the entire time span
+* `points`: the total number of samples that we want, across the entire time span
 
 ## Limitations
 
 This training strategy can only be used with ODEs (`NNODE`).
 """
 struct WeightedIntervalTraining{T} <: AbstractTrainingStrategy
     weights::Vector{T}
-    samples::Int
+    points::Int64
```

Comment:
```suggestion
    points::Int
```

Never use `Int64` unless it's required.